docs/customize/vocabularies/index.md

```diff
@@ -15,6 +15,29 @@ Another behind the scenes note, `invenio-cli services setup` calls `pipenv run i
 !!! warning "Unsupported cases"
     There is no moment between database creation and fixture loading when you can create database entries (e.g., a role) and then use them for your fixtures. However, you can still create roles and assign them to users manually with `invenio` commands. We hope to make loading specific fixtures independent from each other and updating existing fixtures possible in the future.
 
+## Add/Update Fixtures Command
+
+_Introduced in InvenioRDM v12_
+
+To add or update a vocabulary fixture:
+
+```bash
+pipenv run invenio rdm-records add-to-fixture <vocabulary_name>
+```
+
+This command enables the addition or updating of entries within specified vocabulary fixtures in InvenioRDM and is operational on live instances.
+Note that its functionality is restricted to the addition of new vocabularies or the updating of existing ones.
+
+Example
+To update the 'contributorsroles' vocabulary fixture, use:
+
+```bash
+pipenv run invenio rdm-records add-to-fixture contributorsroles
+```
+
+!!! warning "Note"
+    This command will not delete existing vocabularies.
+
```

Comment:
I have 2 questions about the documentation for the `Add/Update Fixtures Command`. 
- This suggests that this command can be used for both adding new vocabularies and updating existing ones. 
Can you confirm if this is indeed the case, or is the command limited to only adding new vocabularies? 
- The placing of this would it be more appropriate to place this section at the end of the page to maintain chronological consistency in the documentation as it's introduced in V12?